src/qmapshack/gis/rte/router/CRouterBRouter.cpp

```diff
@@ -571,7 +568,7 @@ void CRouterBRouter::updateBRouterStatus() const {
     labelStatus->setText(tr("online"));
     toolConsole->setVisible(false);
     toolToggleBRouter->setVisible(false);
-    checkFastRecalc->setVisible(false);
+    labelFastRouting->setText(tr("No auto-routing with BRouter (online)!"));
     textBRouterOutput->clear();
```

Comment:
This is misleading.  You can auto-route but not on-the-fly.  On the other hand side the user is not drawing a route right now and won't understand what this is all about. At least the inexperienced user.

I have to admit that I do not have a good solution or wording right now myself.  Maybe a hint in the tool tip of the "A" button is sufficient. On the other hand side hardly anyone is reading tool tips.  Maybe disabling the "A" button is sufficient. 

Or a message box when the "A" button is pressed. Probably this is the most verbose and bold method.

Reply:
Will remove misleading message totally and realize proposal _Maybe disabling the "A" button is sufficient_.